integrations/localhero/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# Step : package jar
+FROM maven:3.6.3-jdk-11-slim AS build
+RUN mkdir -p /workspace
+WORKDIR /workspace
+COPY pom.xml /workspace
+RUN mvn dependency:go-offline
+
+COPY src /workspace/src
+RUN mvn package -DskipTests
+
+# Step : Package image
+FROM openjdk:11-jdk-slim
```

Comment:
@lugipfupf sorry that I downgraded to version 11 - I wanted to align it with the other service (sync-service) that I already had -> if you have the time, we can of course upgrade both services to the same version

Reply:
Not that it really matters to me. As long as it's not a JEE from aeons ago ;-)